qml/Panel/Panel.qml

```diff
@@ -37,8 +38,7 @@ Item {
 
     property real minimizedPanelHeight: units.gu(3)
     property real expandedPanelHeight: units.gu(7)
-    property real indicatorMenuWidth: width
-    property real applicationMenuWidth: width
+    property real menuWidth: partialWidth ? units.gu(40) : width
```

Comment:
In the old code, there were no application menus. Therefore there was no need to make the indicators get out of the way of the application menus. Your assumption is correct, at least for the touch menus. No matter how wide your screen is, only 40gu is allocated for each menu.

![image](https://user-images.githubusercontent.com/21966173/76375325-87acd200-6313-11ea-9743-ae88a1904b8a.png)

(Shown with rectangles over the touch areas for clarity)

It's done this way because someone decided that the menu width should be the same as the handle width.

![image](https://user-images.githubusercontent.com/21966173/76375357-985d4800-6313-11ea-9e78-c93e3a477b36.png)

Global menus for mouse can take up more of the panel, and fade in when the mouse is over the panel. Similar to Unity7's approach.

I agree this is an issue, but since no known Ubuntu Touch apps use the application menus it should not be a priority right now.